src/main/java/games/strategy/engine/framework/ui/GameChooser.java

```diff
@@ -58,7 +59,7 @@ private void createComponents() {
     notesPanel = new JEditorPane();
     notesPanel.setEditable(false);
     notesPanel.setContentType("text/html");
-    notesPanel.setBackground(new JLabel().getBackground());
+    notesPanel.setForeground(Color.BLACK);
```

Comment:
Fixing the foreground color without also fixing the background color could lead to similar readability problems in other L&Fs that may use a dark background.  Should we consider fixing them both?  Since this panel is basically used to display HTML content, would setting them to match a standard browser's default foreground/background colors of black/white make sense?

A lot of the problems seem to be due to the HTML in the game notes explicitly setting background colors but not foreground colors so that they inherit the default, which the map author probably assumed to be black because they presumably tested their HTML in a browser.

Reply:
Are you suggesting to Change the map xmls to set the foreground color explicitly?

Reply:
Not quite.  I was simply suggesting setting the background on the panel to white.

I have no objection to merging this now, and then we can play around with a few alternatives to see what's the best long-term solution.

Reply:
I don't think setting the background colour to white will do the trick.
It's going to work out for some maps but probably not for all of them. 
So merging...